src/transport/MessageHeader.cpp

```diff
@@ -31,7 +31,6 @@
  * Header format (little endian):
  *
  *  16 bit: | VERSION: 4 bit | FLAGS: 4 bit | RESERVED: 8 bit |
- *  16 bit: | Secure message type                             |
  *  32 bit: | MESSAGE_ID                                      |
  *  32 bit: | Secure Session ID                               |
  *  64 bit: | Message Authentication Tag                      |
```

Comment:
What about support for tags that are 12-bytes and 16-bytes?

Reply:
Do we need support for tags of 12/16 bytes in CHIP? If so, need to update the structure to accommodate those tags.

Reply:
Yes, CHIP supports 3 tag lengths for CCM: 8, 12, and 16
The idea is that tag length is negotiated once during session establishment and only this tag length is used within context of that session.
Few bits of the 16-bit header can be allocated to encode the tag length. However, I might not be necessary because SessionID will identify that only specific tag length can be used for messages sent over that session.

Reply:
Sounds good. Agreed that SessionID can be used to identify what the tag length.